src/test/java/seedu/mtracker/console/GeneralInstrumentParserTest.java

```diff
@@ -7,6 +7,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+//@@author KVignesh122
```

Comment:
I would like to claim the simulateConsoleInput, checkParameters methods here if that is alright with you.

Reply:
Ok have included your github user name in author tag.